### PR TITLE
MM-831 Time change ping riding on the status interupt for timestamp update

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

**Comment by nickago** [comment]
For real this time. The prior code was hitting a conflict with JQuery, props to @jwilander for figuring that out.

File: web/react/components/post_right.jsx [hunk 1]
```diff
@@ -289,11 +291,15 @@ module.exports = React.createClass({
         });
     },
     componentDidUpdate: function() {
-        this.resize();
+        if(!this.wasForced){
+            this.resize();
+            wasForced = false
+        }
```
Comment (ralder) [inline web/react/components/post_right.jsx]: seems it has two typos

Comment (nickago) [inline web/react/components/post_right.jsx]: Good catch on that, update incoming.

Comment (ralder) [inline web/react/components/post_right.jsx]: i aslo missed semicolon :). I mean in condition check `wasForced` is `false` and than `wasForced` set again to `false`

**Comment by rgarmsen2295** [comment]
Just missing two semi-colons but otherwise looks good

**Comment by nickago** [comment]
Revised, ready for re-review

**Comment by ralder** [comment]
github 'outdated diff' hides my message (I don't know it's emailed or not) https://github.com/mattermost/platform/pull/174#discussion_r34497101

**Comment by jwilander** [comment]
@ralder I see that now. Looks like the @nickago fixed the semi-colon but left in the ```this.wasForced = false``` which, you're right, won't do anything since ```this.wasForced``` is already false. @nickago can you submit a new pr to fix that?

Thanks for your help @ralder

**Pull request merged**